bitnami/ghost/5/debian-12/rootfs/opt/bitnami/scripts/ghost-env.sh

```diff
@@ -108,6 +109,8 @@ export GHOST_EMAIL="${GHOST_EMAIL:-user@example.com}" # only used during the fir
 # Ghost SMTP credentials
 GHOST_SMTP_HOST="${GHOST_SMTP_HOST:-"${SMTP_HOST:-}"}"
 export GHOST_SMTP_HOST="${GHOST_SMTP_HOST:-}" # only used during the first initialization
+GHOST_SMTP_FROM_ADDRESS="${GHOST_SMTP_FROM_ADDRESS:-"${SMTP_FROM:-}"}"
+GHOST_SMTP_FROM_ADDRESS="${GHOST_SMTP_FROM_ADDRESS:-"${GHOST_SMTP_FROM_ADDRESS:-}"}"
```

Comment:
I think we can keep using "GHOST_EMAIL" as default value and modify it if the user provides a different one. Sounds good? 

```suggestion
GHOST_SMTP_FROM_ADDRESS="${GHOST_SMTP_FROM_ADDRESS:-"${GHOST_EMAIL:-}"}"
```

Reply:
Yeah, that makes sense. Removes a breaking change and ensures a sane default.